Multisig/UI/UI Library/TopTabView/BottomBorder.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  BottomBorder.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 14.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+// Draws a configurable-height bottom border
+struct BottomBorder: Shape {
```

Comment:
BottomBorder is not clear here, usually they call it "selector" or so

Reply:
`Underline`?

Reply:
I still prefer selector, as it's indicate the selection state

Reply:
but it is a shape, I would name it selector if it would be a view, then it would have a clear purpose, but the shape is very generic, it can be selector in one place, and a decoration in another place (let's say underline of a text field)

Reply:
and bottom border is saying what the shape is, and it is clear what will be drawn

Reply:
Both ways are fine I think